Maestro/subscriptions.json

```diff
@@ -18,18 +18,24 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
-    // A 'build' that mirrors changes from github into the devdiv intance of VSTS
-    "github-devdiv-vsts-mirror": {
+    // A 'build' that mirrors changes from github into the devdiv intance of Azure DevOps
+    "github-devdiv-azdo-mirror": {
       "vsoInstance": "devdiv.visualstudio.com",
       "vsoProject": "DevDiv",
       "buildDefinitionId": 8289
     },
-    // A 'build' that mirrors changes from github into the dnceng intance of VSTS
-    "github-dnceng-vsts-mirror": {
+    // A 'build' that mirrors changes from github into the dnceng intance of Azure DevOps
+    "github-dnceng-azdo-mirror": {
       "vsoInstance": "dnceng.visualstudio.com",
       "vsoProject": "internal",
       "buildDefinitionId": 16
     },
+    // A 'build' that mirrors changes from github into the dnceng intance of Azure DevOps
+    "github-dnceng-azdo-internal-merge-mirror": {
```

Comment:
From the PR title I can guess what it does, but the def names and comments seem mismatched making the diff confusing. Is this similar to `dotnet-devdiv-branch-merge-mirror`, should one of the names be clarified to represent that?

Reply:
Updated